### PR TITLE
Iteration on SMP to better scale by size of software project

### PR DESCRIPTION
**Pull request by yvgrossmann**
Done with @jwindeck during the NFDI4DataScience Hackathon maSMPs in November 2023, organised by @ljgarcia at the ZB Med

New elements:

| Type      | Key                                 | Details    |
| --------- | ----------------------------------- | ---------- |
| Attribute | application-class                   |            |
| Optionset | application-class                   | 4 options  |
| Optionset | coding-standards                    | 5 options  |
| Optionset | software_identifier                 | 4 options  |
| Optionset | software-license                    | 8 options  |
| Optionset | testing                             | 10 options |
| Condition | application-class-1                 |            |
| Condition | application-class-2                 |            |
| Condition | application-class-3                 |            |
| Question  | smp/general/topic/application-class |            |

### CODE REVIEW

**Comment by yvgrossmann** [comment]
This activity was carried out during the maSMP hackathon at [ZB MED](https://www.zbmed.de/en/) sponsored by [NFDI4DataScience](https://www.nfdi4datascience.de/). NFDI4DataScience is a consortium funded by the German Research Foundation (DFG), project number 460234259.

**Comment by yvgrossmann** [comment]
@jwindeck and me added software license as controlled voc to support the cff plugin in RDMO

File: rdmorganiser/questions/SMP_Readme.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ The catalogue uses the generic elements from RDMO. Therefore, all generic elemen
 This catalogue introduces many custom attributes, as the generic RDMO attributes do not always cover the specific aspects of software. For this reason, the existing views in RDMO cannot be used for this catalogue, or only to a very limited extent.
 
 # Use outside RDMO
-Due to multiple requests, we have created a template on the SMP for use outside of RDMO. The Word document is freely writable, has a CC0 and is freely available via https://doi.org/10.17617/2.3481986  (direct link to the docx. file: https://hdl.handle.net/21.11116/0000-000C-1076-D) for your own re-use.
+Due to multiple requests, we have created a template on the SMP for use outside of RDMO. The Word document is freely writable, has a CC0 and is freely available via https://doi.org/10.17617/2.3481986  (direct link to the docx. file: https://hdl.handle.net/21.11116/0000-000C-1076-D) for your own re-use. A metadata crosswalk document is published via https://doi.org/10.5281/zenodo.10275895.
```
Comment (Zack-83) [inline rdmorganiser/questions/SMP_Readme.md:34]:
```suggestion
Due to multiple requests, we have created a standalone SMP template for use outside of RDMO. The Word document is freely available under a CC0 license via https://doi.org/10.17617/2.3481986 (direct link to the docx file: https://hdl.handle.net/21.11116/0000-000C-1076-D) for your own re-use.

# Metadata crosswalk to the maSMP standard
The attributes used to create an SMP have been aligned to the recommendations on acknowledge machine-actionable software management plans (maSMPs) being developed in the German SMP community. A metadata crosswalk document is published on https://doi.org/10.5281/zenodo.10275895.
```

**Review by Zack-83 — APPROVED** [review]

File: rdmorganiser/questions/SMP-Questions.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<rdmo created="2023-08-18T14:53:21.777351+02:00" xmlns:dc="http://purl.org/dc/elements/1.1/">
+<rdmo xmlns:dc="http://purl.org/dc/elements/1.1/" created="2023-11-29T16:10:14.740331+01:00" version="3.0.0">
```
Comment (Zack-83) [inline rdmorganiser/questions/SMP-Questions.xml:2]:
```suggestion
<rdmo xmlns:dc="http://purl.org/dc/elements/1.1/" created="2023-11-29T16:10:14.740331+01:00">
```

**Pull request merged**